otx/algorithms/detection/adapters/mmdet/models/heads/custom_atss_head.py

```diff
@@ -172,18 +172,16 @@ def loss_single(
             pos_centerness = centerness[pos_inds]
 
             centerness_targets = self.centerness_target(pos_anchors, pos_bbox_targets)
-            pos_decode_bbox_pred = self.bbox_coder.decode(pos_anchors, pos_bbox_pred)
-            pos_decode_bbox_targets = self.bbox_coder.decode(pos_anchors, pos_bbox_targets)
+            if self.reg_decoded_bbox:
```

Comment:
This change looks like a commit that has already been applied to develop. Please check your commits as they may be mixed up.

Reply:
Revised. Thanks:)